cmd/bucket-replication.go

```diff
@@ -1612,9 +1627,21 @@ func proxyHeadToRepTarget(ctx context.Context, bucket, object string, opts Objec
 			Internal: miniogo.AdvancedGetOptions{
 				ReplicationProxyRequest: "true",
 			},
+			PartNumber: opts.PartNumber,
 		}
+		if rs != nil {
+			h, err := rs.ToHeader()
+			if err != nil {
+				continue
+			}
+			gopts.Set(xhttp.Range, h)
```

Comment:
let's not ignore this header and log it just in case since it is actionable to the user @poornas